README.md

```diff
@@ -510,7 +509,6 @@ A collection of awesome things regarding React ecosystem.
 * [react-responsive - Media queries in react for responsive design](https://github.com/wearefractal/react-responsive)
 * [react-cursor - Functional state management abstraction for use with Facebook React](https://github.com/dustingetz/react-cursor)
 * [Omniscient.js - Abstracting React components for fast top down render of immutable data](https://github.com/omniscientjs/omniscient)
-* [Touchstonejs - React.js powered UI framework for developing beautiful hybrid mobile apps.](http://touchstonejs.io/)
```

Comment:
Instead of removing this, how about replacing it with the working link?

https://github.com/touchstonejs/touchstonejs